packages/components/src/table/index.js

```diff
@@ -453,6 +460,14 @@ TableCard.propTypes = {
 	 * A function which returns a callback function which is called upon the user changing the visiblity of columns.
 	 */
 	onColumnsChange: PropTypes.func,
+	/**
+	 * A function which is called upon the user searching in the table header.
+	 */
+	onSearch: PropTypes.func,
```

Comment:
With the addition of these two new props, I think we should note the changes in the "unreleased" section of the components package.

Reply:
Done in 7030953.